src/javascript/core/user.js

```diff
@@ -24,10 +24,14 @@ const Store = require('./store');
  * @param {String} value The value of a userID to use if one is not stored
  * @return {String} - The user ID if present, or a generated UID if not
```

Comment:
Add as a `@param` here as well if that's the pattern so far?

Reply:
(I also feel that perhaps it'd be more useful to pass the whole `config` object in here for future use...?)

Reply:
Yes this would probably benefit from being the whole config. I think throughout the rest of the code base, config is handled via the settings file. Unfortunately in the current iteration the user / store gets initialised before the config is saved to the settings file and so this change would require a bigger overhaul.